CHANGELOG.md

```diff
@@ -44,6 +44,7 @@
 * [ENHANCEMENT] Disabled in-memory shuffle-sharding subring cache in the store-gateway, ruler and compactor. This should reduce the memory utilisation in these services when shuffle-sharding is enabled, without introducing a significantly increase CPU utilisation. #3601
 * [ENHANCEMENT] Shuffle sharding: optimised subring generation used by shuffle sharding. #3601
 * [ENHANCEMENT] New /runtime_config endpoint that returns the defined runtime configuration in YAML format. The returned configuration includes overrides. #3639
+* [ENHANCEMENT] query_range API to include parameter name in HTTP 400 message. #3699
```

Comment:
A couple of suggestions:
1. We're used to link the PR number instead of the issue number
2. If the change affects only 1 specific Cortex service then we prefix it

```suggestion
* [ENHANCEMENT] Query-frontend: included the parameter name failed to validate in HTTP 400 message. #3703
```

Reply:
Thanks for the suggestion and fixing it for me.